source/val/validate_extensions.cpp

```diff
@@ -1082,6 +1082,23 @@ spv_result_t ValidateExtInstImport(ValidationState_t& _,
   return SPV_SUCCESS;
 }
 
+spv_result_t ValidateExtInstWithForwardRefs(ValidationState_t& _,
```

Comment:
I'm not certain this function is needed given most of the checks are handled in validate_id.cpp. Maybe just that the import instruction is indeed an OpExtInstImport.

Reply:
removed and tests are still fine, was redundant.